.changeset/proud-yaks-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/codemod': patch
+---
+
+feat (packages/codemod): Add script to scaffold new codemod.
```

Comment:
internals dev changes might not need a release, so no changeset would have been needed. (no need to remove tho)